drivers/SmartThings/philips-hue/src/init.lua

```diff
@@ -511,7 +511,7 @@ local function init_light(driver, device)
       device:set_field(Fields.MIN_KELVIN, HueApi.MIN_TEMP_KELVIN_WHITE_AMBIANCE, { persist = true})
     end
   end
-  local device_light_resource_id = device:get_field(Fields.RESOURCE_ID) or device.device_network_id
+  local device_light_resource_id = device:get_field(Fields.RESOURCE_ID) or device.parent_assigned_child_key
```

Comment:
do we need to consider old device's `device.device_network_id` here?

Reply:
Yes, good eyes.